tests/integration/records/test_records_serializers_json.py

```diff
@@ -148,21 +148,58 @@ def test_zero_citations_in_vnd_plus_inspire_record_ui_json(isolated_api_client):
                                        headers={'Accept': 'application/vnd+inspire.record.ui+json'})
     assert response.status_code == 200
     result = json.loads(response.get_data(as_text=True))
-    assert result['metadata']['citations_count'] == 0
+    assert result['metadata']['citation_count'] == 0
 
 
 def test_non_zero_citation_count_in_vnd_plus_inspire_record_ui_json(isolated_api_client):
+    from invenio_db import db
+    with db.session.begin_nested():
+        record_json = {
+            'control_number': 123,
+        }
+        record = TestRecordMetadata.create_from_kwargs(json=record_json).inspire_record
+        url = '/literature/123'
+        # Add citation
+        ref = {'control_number': 1234, 'references': [{'record': {'$ref': record._get_ref()}}]}
+        TestRecordMetadata.create_from_kwargs(json=ref)
+
+        response = isolated_api_client.get(url,
+                                           headers={'Accept': 'application/vnd+inspire.record.ui+json'})
+        assert response.status_code == 200
+        result = json.loads(response.get_data(as_text=True))
+        assert result['metadata']['citation_count'] == 1
+
+        # # Add second citation
+        # ref = {'control_number': 12341, 'references': [{'record': {'$ref': record._get_ref()}}]}
+        # TestRecordMetadata.create_from_kwargs(json=ref)
+        #
+        # response = isolated_api_client.get(url,
+        #                                    headers={'Accept': 'application/vnd+inspire.record.ui+json'})
+        # assert response.status_code == 200
+        # result = json.loads(response.get_data(as_text=True))
+        # assert result['metadata']['citation_count'] == 2
```

Comment:
Leftovers? :)

Reply:
Not really, but may be better if I will remove them.